src/main/java/seedu/duke/Duke.java

```diff
@@ -78,6 +126,73 @@ private static void addAnime() {
         System.out.println("===End of Sample Anime Class===");
     }
 
+    private static void createWatchlist(String description) {
+        String[] descriptionSplit = description.split(" ", 2);
+        String commandOption = descriptionSplit[0];
```

Comment:
It's not used at this point yet, perhaps we can use a if statement for now since there be more options later.
```
if (commandOption.equals("-n")) {
    // create watchlist code
}
```